swagger-spec/licenses/detail.yaml

```diff
@@ -1 +1,29 @@
-# /licenses/{license_id}/
+parameters:
+  - in: path
+    type: string
+    required: true
+    name: license_id
+    description: 'The unique identifier of the license.'
+
+get:
+  summary: Retrieve a license
+  description: >-
+    Retrieves the details of a license.
+
+    #### Returns
+
+    Returns a dictionary with a `data` property containing the representation of the requested
+    license, if the request is successful.
+
+
+    If the request is unsuccessful, a dictionary with an `errors` property containing
+    information about the failure will be returned. Refer to the [list of error codes]()
+    to understand why this request may have failed.
+  tags:
+    - Licenses
+  operationId: licenses_read
+  responses:
+    '200':
+      description: 'OK'
```

Comment:
An example response can be added here (similar to the example in a below comment, minus the pagination links part of the response since it's a detail view):

```
examples:
  application/json:
    <response>
```